api/models/Message.js

```diff
@@ -1,7 +1,7 @@
 const mongoose = require('mongoose');
 
 const messageSchema = mongoose.Schema({
-  id: {
+  messageId: {
```

Comment:
This is a breaking change. I understand it but prior messages are now unavailable

Reply:
yes. I'm new to mongoose. do they have some migration script? like django orm.